src/components/projects-summary/projects-summary.html

```diff
@@ -30,7 +30,7 @@ <h4 class="catalog-modal-title">
     </a>
   </div>
   <div ng-if="$ctrl.projects && $ctrl.projects.length" class="catalog-project-summary-list">
-    <div ng-if="$ctrl.totalProjects > $ctrl.maxDisplayProjects" class="projects-count">
+    <div class="projects-count">
       <strong>{{$ctrl.maxDisplayProjects}}</strong>
       of
       <strong>{{$ctrl.totalProjects}}</strong>
```

Comment:
@beanh66 Should we be showing "3 of 3 Projects" in addition to the view all link?